falcon/request.py

```diff
@@ -838,6 +838,44 @@ def get_param_as_list(self, name,
 
         raise HTTPMissingParam(name)
 
+    def get_param_as_date(self, name, format_string='%Y-%m-%d',
+                          required=False, store=None):
+        """
+        Args:
+            name (str): Parameter name, case-sensitive (e.g., 'ids').
+            format_string (str): String used to parse the param value into a
+date.
```

Comment:
Looks like you'll need to indent this line to make sphinx happy (see attached). You can make the docs to check your changes:

```shell
pip install -r tools/doc-requires
cd doc
make html
open _built/html/index.html
```

![screen shot 2015-04-15 at 10 53 30 am](https://cloud.githubusercontent.com/assets/719939/7161707/798e9116-e35e-11e4-9a7f-9c10ed3e0643.png)